dask_cuda/local_cuda_cluster.py

```diff
@@ -166,6 +171,16 @@ def __init__(
         self.host_memory_limit = memory_limit
         self.device_memory_limit = device_memory_limit
 
+        self.rmm_pool_size = rmm_pool_size
+        if rmm_pool_size is not None:
+            try:
+                import rmm
+            except ImportError:
+                raise ImportError(
+                    "RMM pool requested but module 'rmm' is not available"
+                )
```

Comment:
Should make coverage happier. Not sure if we need this for all lines or just the last one.

```suggestion
                )  # pragma: no cover
```

Reply:
Does that also work for codecov? I'm not sure whether it uses coverage.py or the same syntax.

Reply:
Yeah I think all Python coverage tools use `coverage.py`.

Reply:
Thanks @jakirkham , it seems to work indeed!